tools/lint

```diff
@@ -89,7 +89,8 @@ def run() -> None:
                                   description="Checks commit messages for common formatting errors."
                                   "(config: .gitlint)")
 
-    semgrep_command = ["semgrep", "--config=./tools/semgrep.yml", "--error"]
+    semgrep_command = ["semgrep", "--config=./tools/semgrep.yml", "--error",
+                       "--dangerously-allow-arbitrary-code-execution-from-rules"]
```

Comment:
So, this certainly sounds like a dangerous option.  Is there any detail from the maintainers on what it means?  This at very least deserves a comment explaining why it's not dangerous for us (which it may very well be -- they probably care about this from the context of running this as a service for customeres).

Reply:
Added a comment for this in the code. It should be fine as long as the python snippet we run in there isn't harmful in any way. 
See docs: https://github.com/returntocorp/semgrep/blob/experimental/docs/config/advanced.md#pattern-where-python